jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/DeploymentDescriptorEditorPresenter.java

```diff
@@ -239,16 +246,9 @@ public Menus getMenus() {
     }
 
     protected void makeMenuBar() {
-        menus = menuBuilder
-                .addSave( versionRecordManager.newSaveMenuItem(new Command() {
-                    @Override
-                    public void execute() {
-                        onSave();
-                    }
-                }))
+        fileMenuBuilder
```

Comment:
@nmirasch There was an issue with subclasses of ```KieEditor``` masking the "Menu Builder" which leads to menus not being shown for the editor. This change removes the masking.